source/dotnet/Library/AdaptiveCards/AdaptiveHeightConverter.cs

```diff
@@ -0,0 +1,58 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+using System;
+using System.Collections.Generic;
+using System.Globalization;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+
+namespace AdaptiveCards
+{
+    internal class AdaptiveHeightConverter : JsonConverter<AdaptiveHeight>, ILogWarnings
+    {
+        public List<AdaptiveWarning> Warnings { get; set; } = new List<AdaptiveWarning>();
+
+        public AdaptiveHeightConverter()
+        {
+        }
+
```

Comment:
super nit: extra space

Reply:
not entirely clear from the diff given here, but I was referring to line 19 :D

Reply:
done